spec/acceptance/sensu_bonsai_asset.rb

```diff
@@ -60,6 +60,10 @@
         version  => 'latest',
         provider => 'sensu_api',
       }
+      sensu_bonsai_asset { 'sensu/sensu-go-has-contact-filter':
```

Comment:
How does this test show us anything?

Reply:
The condition of failure was having multiple bonsai assets defined having different latest versions in Bonsai, using the same provider. The previous tests had one defined via `sensuctl` provider and one via `sensu_api` provider.  Having two resources using `sensuctl` would use the latest version from one and incorrectly assume it was the latest version for other resources of same provider.  This is a more direct test that was detected in contact routing example with same resources.

Reply:
Could you please put a comment in the code to explain this and link to this issue.

Reply:
Comment added.